app/models/file_upload.rb

```diff
@@ -0,0 +1,7 @@
+class FileUpload < ApplicationRecord
+  belongs_to :user
```

Comment:
It would be nice to add a quick new feature for a simple page showing a users uploads along with the status (if the job is pending, if there were errors), as the data is all there now.  This would replace the existing Uploads > File List